sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -549,11 +549,16 @@ def my_pipeline():
         self.assertEqual(template['retryStrategy']['backoff']['maxDuration'],
                          backoff_max_duration)
 
-    def test_py_runtime_memory_request(self):
-        """Test memory request."""
-
-        def my_pipeline(memory: str, cpu: str):
-            some_op().set_cpu_request(memory)
```

Comment:
looks like there were a discrepancy between the test name and what it was actually testing (confusion between memory & cpu) ?